plugins/hydra_nevergrad_sweeper/hydra_plugins/hydra_nevergrad_sweeper/core.py

```diff
@@ -0,0 +1,220 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+import json
+import logging
+from typing import Any, Dict, List, Optional, Union
+
+import nevergrad as ng  # type: ignore
+from omegaconf import DictConfig, OmegaConf
+
+from hydra.core.config_loader import ConfigLoader
+from hydra.core.config_search_path import ConfigSearchPath
+from hydra.core.plugins import Plugins
+from hydra.plugins.launcher import Launcher
+from hydra.plugins.search_path_plugin import SearchPathPlugin
+from hydra.plugins.sweeper import Sweeper
+from hydra.types import TaskFunction
+
+log = logging.getLogger(__name__)
+
+
+class NevergradSearchPathPlugin(SearchPathPlugin):
+    def manipulate_search_path(self, search_path: ConfigSearchPath) -> None:
+        search_path.append(
+            "nevergrad", "pkg://hydra_plugins.hydra_nevergrad_sweeper.conf"
```

Comment:
Potential followup after this lands:
Take a look at the config of the [Joblib plugin](https://hydra.cc/docs/next/plugins/joblib_launcher/), I ported it to use Structured configs and it's much cleaner.
This will require learning about structured configs from the [new tutorial](https://hydra.cc/docs/next/tutorials/structured_config/basic).

Reply:
created #478.